iphone/Maps/UI/EditBookmark/EditTrackViewController.swift

```diff
@@ -0,0 +1,180 @@
+//
+//  EditTrackViewController.swift
+//  OMaps
+//
```

Comment:
@vng @biodranik do we have a standardized verbatim for sources? I'd prefer to follow existing style and don't add the header.

Reply:
Not yet, but here these useless comments should be removed.

Reply:
Header comment removed.